chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -106,6 +110,10 @@
       done
     EOF
   end
+  # Deletes postgres data dir in non-HA scenario
+  directory "/var/lib/pgsql/data" do
+    action :delete
```

Comment:
@mjura In the UI we tell the user to download the database dump, we could also give him the opportunity to grab a copy of that data dir (it would require some rearrangements though). What would be the reason to keep the data dir on the device as well? One reason against it is disk space issues.

@vuntz I am a somewhat undecided on this, could you please be the tie-breaker ;)